tests/test_rex.py

```diff
@@ -40,7 +40,7 @@ def test_legit_00001():
     for leaker in arsenal.leakers:
         nose.tools.assert_true(_do_pov_test(leaker))
 
-def test_legit_00003():
+def test_legit_00003():#L51 + 52
```

Comment:
Can you explain what these added numbers are?

Reply:
Oh, this is the line number where it fails. I want to leave the number there until it is fixed. Except that if it bothers you, I can remove it